config/routes.rb

```diff
@@ -4,7 +4,7 @@
   namespace :api do
     namespace :v1 do
       resource :health, only: [:show], controller: :health
-      resource :profile, only: [:show], controller: :profile
+      resource :profile, only: [:show]
```

Comment:
To keep this more RESTful, I think this resource should be plural, not singular
```suggestion
      resources :profiles, only: [:show]
```